Custom Model Client docs follow-up (#1545)
microsoft/autogen: website/docs/Use-Cases/enhanced_inference.md

For local LLMs, one can spin up an endpoint using a package like [FastChat](https://github.com/lm-sys/FastChat), and then use the same API to send a request. See [here](/blog/2023/07/14/Local-LLMs) for examples on how to make inference with local LLMs.

> For custom model clients, one can register the client with `autogen.OpenAIWrapper.register_model_client` and then use the same API to send a request. See [here](blog/2024/01/26/Custom-Models) for examples on how to make inference with custom model clients.

Reviewer: ```suggestion
For custom model clients, one can register the client with `autogen.OpenAIWrapper.register_model_client` and then use the same API to send a request. See [here](/blog/2024/01/26/Custom-Models) for examples on how to make inference with custom model clients.
```

* Model 'gpt-3.5-turbo': cost: 0.00027, prompt_tokens: 50, completion_tokens: 100, total_tokens: 150
```

> Note: if using a custom model client (see [here](blog/2024/01/26/Custom-Models) for details) if usage summary is not implemented, then the usage summary will not be available.

Reviewer: ```suggestion
Note: if using a custom model client (see [here](blog/2024/01/26/Custom-Models) for details) and if usage summary is not implemented, then the usage summary will not be available.
```

> Note: if using a custom model client (see [here](blog/2024/01/26/Custom-Models) for details) and if usage summary is not implemented, then the usage summary will not be available.

Reviewer: ```suggestion
Note: if using a custom model client (see [here](/blog/2024/01/26/Custom-Models) for details) and if usage summary is not implemented, then the usage summary will not be available.
```

        {
            "model": "llama2-chat-7B",
            "base_url": "http://127.0.0.1:8080",
        },
        {
            "model": "microsoft/phi-2",
            "model_client_cls": "CustomModelClient"
        }
    ],
)
```

> `client.create()` will try querying Azure OpenAI gpt-4, OpenAI gpt-3.5-turbo, a locally hosted llama2-chat-7B, and phi-2 using custom provided inference, one by one,

Reviewer: ```suggestion
`client.create()` will try querying Azure OpenAI gpt-4, OpenAI gpt-3.5-turbo, a locally hosted llama2-chat-7B, and phi-2 using a custom model client class named `CustomModelClient`, one by one,
```
